src/client/components/app/Transactions/components/TxTTLInput.tsx

```diff
@@ -153,9 +153,9 @@ export const TxTTLInput: FC<TxTokenInputProps> = ({
           <AmountTitle ellipsis>{inputText || t('components.transaction.token-input.you-have')}</AmountTitle>
           <AmountInputContainer>
             <StyledAmountInput
-              value={ttlType ? +amount / 60 / 60 / 24 : +amount}
+              value={amount}
```

Comment:
is `ttlType` not used any more?

Reply:
No, I moved the logic up to the parent component, that shouldn't have been there.